containers/server-image/Dockerfile

```diff
@@ -60,9 +62,7 @@ RUN zypper --gpg-auto-import-keys --non-interactive install --auto-agree-with-li
     ipmitool \
     sssd \
     sssd-dbus \
-    sssd-ldap \
-    nss-pam-ldapd \
-    authselect
```

Comment:
couldn't you add those two packages to the EXTRA_PACKAGES argument as default value?